cli/launch.template.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
This was initially going to be a pair of shell scripts. However, I found that extracting named arguments without removing all the others (since we want to pass them to Java) is a bit tricky. Furthermore, Bash and Batch scripts have a completely different approaches to argument processing.

As I've tried out the Python version, I found it to be more readable as well. That is, of course, subjective.

As an objective benefit, we have to support only one script file, not two.
Given that most engineers are going to have Python already installed, I think it's OK to run it in Python.